composer.json

```diff
@@ -9,9 +9,10 @@
             "homepage": "https://github.com/equip"
         }
     ],
+    "minimum-stability": "alpha",
     "require": {
         "php": ">=5.5",
-        "equip/adr": "^1.3",
+        "equip/adr": "^2.0",
```

Comment:
Released `2.0.0-alpha1` to ensure that deprecated constants are not used in 2.0.